src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -9,5 +9,7 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MAIL_APP_ERROR = "Error opening the default mail app on this system";
```

Comment:
Follow the variable naming standard used above. Use names like MESSAGE_MAIL_APP_ERROR